Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -1,76 +1,138 @@
 
-_Maximum Produced Torque:_
+### Constraints:
 
-Based on a study conducted by R C Haut [6], the force required to fracture the patella or femur is approximately 8.5 kN. Applying the gear ratio of the Mario Kart bike and the radius of its wheel, we come to a torque of 4748.94 Nm (torque = gear ratio * wheel radius * force) required to fracture these leg bones. Creating an upper limit of 10% of this value, we get an upper limit torque of 474.89 Nm. Our produced torque utilizing the eddy current braking system shall not exceed this value to protect the user from bodily harm from changes in resistance.
+*Minimum Flywheel Speed:*
 
-**_Buildable schematic:_**
+•	While the user is pedaling the bike at minimum speed, the resistance system must be capable of varying the torque, which will result in the varying of resistance experienced by the rider.
 
-_Resistance System:_
+•	The specified minimum speed of 116.35 rpm is based on the American Council on Exercise's recommendation for the minimum speed required for moderate exercise on a bike [2].
 
-![image](https://user-images.githubusercontent.com/114370750/204955948-41f69d9c-ff85-43b6-aaa1-f8d9cf343a95.png)
+•	At speeds less than 116.35 rpm, the performance of the system will not adhere to any non-safety constraints.
 
-Figure 1. 3D Model Of Flywheel, Magnet, and Bracket
+*Maximum Flywheel Speed:*
 
-![image](https://user-images.githubusercontent.com/114370750/204956305-c4b9ecdb-e11f-4258-981c-f0aa6780823b.png)
+•	The maximum speed of the bike will be set to 288 rpm (bike tire) to restrict users from any personal injury while riding. 
 
-Figure 2. Magnet Dimensions
+•	The maximum speed is based on the pedaling rpm of 118, which is equal to that of an elite-level cyclist [3].
 
-![image](https://user-images.githubusercontent.com/114370750/204956402-5a1c328c-9039-4abf-8c40-277091f3db32.png)
+•	The maximum speed was chosen due to it being a good limit for high speeds, as the normal user will not be able to pass it.
 
-Figure 3. Flywheel Dimensions
+•	When the user is traveling equal to or faster than 288 rpm, the resistance system will not apply any additional braking torque to the flywheel to avoid any injury to the rider.
 
-![image](https://user-images.githubusercontent.com/114370750/204956463-ca256803-743d-4289-af42-5ac5a970c525.png)
+*Maximum Produced Torque:*
 
-Figure 4. Bracket Dimensions
+•	The potential resistance generated (braking torque) will increase as the bike tire speed increases, resulting in the need to restrict the maximum amount of resistance felt by the user. 
 
+•	Based on a study conducted by R C Haut [4], the force required to fracture the patella or femur is approximately 8.5 kN. 
 
-**_Analysis:_**
+•	Applying the gear ratio of the bike and the radius of its wheel, a torque of 4748.94 N*m would be required to injure a rider.
 
+•	The maximum amount of resistance generated at 288 rpm (max bike speed) will be 71.5 N*m.
```

Comment:
Based on what? at what distance? These considerations seem good, but they seem to be out of thin air.